caf-logging-serilog/README.md

```diff
@@ -0,0 +1,67 @@
+# CAF Logging Serilog
+This project configures Serilog to meet the CAF Logging Standard.
+
+The log level is controllable via value that can be provided on Logger initialization. The default used is `INFO`.
```

Comment:
Can't we control the log level by setting the `CAF_LOG_LEVEL` environment variable the same way we do with the Logback and the Log4j 2 versions?

Reply:
Done